src/components/ebay-video/versions.json

```diff
@@ -1 +1 @@
-{ "//": "This is a generated file. Run script file to update", "shaka": "4.2.2" }
+{ "//": "This is a generated file. Run script file to update", "shaka": "4.3.0" }
```

Comment:
How come this video file got an update from the 3d viewer change?

Reply:
I mentioned that I updated it since it was out of date. I had not updated it earlier. 
The CDN generate script updates those version files so it tells the package which version to use. Since we have two files in CDN (shaka and media viewer), it got updated as well.